01_login.php

```diff
@@ -34,9 +34,10 @@
 	<head>
 		<title>Sound Wave</title>
 		<meta charset = "utf-8">
+
+		<!-- styles -->
 		<link rel = "stylesheet" type = "text/css" href = "css/fonts.css">
 		<link rel = "stylesheet" type = "text/css" href = "css/grids.css">
-		<link rel = "stylesheet" href = "css/style.css"/>
```

Comment:
deleted style sheet; due to having two others called fonts and grids. There was no need for another file.